modules/okenetwork/locals.tf

```diff
@@ -37,6 +37,14 @@ locals {
 
   # control plane
   cp_egress = [
+    {
+      description      = "Allow Kubernetes control plane to communicate with other services on this subnet (bastion service)",
+      destination      = local.cp_subnet,
+      destination_type = "CIDR_BLOCK",
+      protocol         = local.tcp_protocol,
+      port             = -1,
```

Comment:
This should be port 6443

Reply:
I tried it with 6443 and it did not work.  I believe this is because it is an egress rule and the traffic is not seen as 'coming from' 6443.